docs/features/search/how-to-guides.md

```diff
@@ -97,3 +97,37 @@ of the `SearchType` component.
   ...
 </Paper>
 ```
+
+## How to limit what can be searched in the Software Catalog
+
+The Software Catalog includes a wealth of information about the components,
+systems, groups, users, and other aspects of your software ecosystem. However,
+you may not always want _every_ aspect to appear when a user searches the
+catalog. Examples include:
+
+- Entities of kind `Location`, which are often not useful to Backstage users.
+- Entities of kind `User` or `Group`, if you'd prefer that users and groups be
+  exposed to search in a different way (or not at all).
+
+It's possible to write your own [Collator](./concepts.md#collators) to control
+exactly what's available to search, (or a [Decorator](./concepts.md#decorators)
```

Comment:
Do you think its valuable to give an example of how to implement a new collator/decorator here? Like another how to guide. Could be done in a separate PR though!

Reply:
For sure!  But maybe after #7239..... 😬